staging/src/k8s.io/client-go/examples/in-cluster-client-configuration/README.md

```diff
@@ -37,7 +37,7 @@ kubectl create clusterrolebinding default-view --clusterrole=view --serviceaccou
 
 Then, run the image in a Pod with a single instance Deployment:
 
-    kubectl run --rm -i demo --image=in-cluster --image-pull-policy=Never
+    kubectl run --rm -i demo --image=in-cluster --image-pull-policy=IfNotPresent
```

Comment:
Since `IfNotPresent` is the default, just omit this flag entirely and simplify this example?

Reply:
Sounds good, will do that!